### DISCUSSION TITLE
加缪 | /cn/2019/02/camus/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1327] [Comments]
https://yihui.org/cn/2019/02/camus/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
谢谢推荐，这本书我已经放在书架上。

> ——原帖发布于 2019-07-22 03:47:37

[a comment by an automated account is not shown]

**Comment by residualsun1** [comment]
> 时常念叨自己弱点的人，到底是想改掉它，还是纯粹只是想受到怜悯和鼓励呢？

看得我心里也是一阵咯噔，分享看过的[一段话](https://mp.weixin.qq.com/s/Th5a3q9owoNs9jd2c3TOnQ)，都给了我拷问的感觉：

> 我并不否认真正的悲伤情绪，但我认为“现代的悲伤=实质悲伤+表演悲伤”，我们在无意识中利用消极情绪达成某些目的，而在这个过程中主体的渲染使悲伤看起来更加沉重。这个逻辑使我们不得不承认，即使只是在极少数的情况中，这种“伤感”的本质是自负与自恋的。正如保罗·德曼对卢梭《忏悔录》的解构：卢梭的忏悔用虚假的欲望代替了真实的欲望，消解了其忏悔的严肃性，其忏悔话语本质上是辩解话语。当我们开始宣泄和使用消极情绪或者刻意将自己放在一个很低的位置的时候，我们一定要问自己一句，你的目的是什么？
